radixdlt-engine/src/main/java/com/radixdlt/application/system/scrypt/SystemConstraintScrypt.java

```diff
@@ -157,20 +154,7 @@ public ReducerState claim(UnclaimedREAddr unclaimedREAddr, ExecutionContext ctx)
 
   @Override
   public void main(Loader os) {
-    os.substate(
-        new SubstateDefinition<>(
-            VirtualParent.class,
-            SubstateTypeId.VIRTUAL_PARENT.id(),
-            buf -> {
-              REFieldSerialization.deserializeReservedByte(buf);
-              var data = new byte[buf.remaining()];
-              buf.get(data);
-              return new VirtualParent(data);
-            },
-            (s, buf) -> {
-              REFieldSerialization.serializeReservedByte(buf);
-              buf.put(s.data());
-            }));
+    os.substate(VirtualParent.SUBSTATE_DEFINITION);
 
     // TODO: Down singleton
     os.procedure(
```

Comment:
Can we do the same for the procedures?

Reply:
I have a huge temptation to do so, but so far, this goes way beyond the scope of the ticket.